src/main/resources/db_scripts/db_script_v3.sql

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
Yes you are right, but now we can see whole table structure without going over v0 and all delta scripts. Maybe it would make sense to have v0 and vLast. (copy-paste risks is still there but is lower)

Reply:
Ah that's the reason. I think it's better to look at the DB directly than relying on these scripts because any typo, copy-paste error will go unnoticed. But ok, we can discuss about it in the context of the liquibase feature.

Reply:
ok